content/dev/release_process/release_tagging.md

```diff
@@ -58,12 +58,13 @@ For Pre-Releases:
 - Don't forget to check the pre-release box 
 
 For Official Releases: 
+- Flick the "Generate Release Notes" button to auto-generate a list of changes and contributors
```

Comment:
"Flick"? :D

Reply:
It's French for "click." :)
Will fix on next checkin.